packages/test-ui/src/components/Swap.tsx

```diff
@@ -590,8 +585,8 @@ export const Swap = ({ web3Provider, signer, chainData }: SwapProps): ReactEleme
                 console.log("changed: ", changed);
               }}
               initialValues={{
-                sendingChain: getChainName(parseInt(Object.keys(swapConfig[selectedPoolIndex].assets)[0]), chainData),
-                receivingChain: getChainName(parseInt(Object.keys(swapConfig[selectedPoolIndex].assets)[1]), chainData),
+                sendingChain: Object.keys(swapConfig[selectedPoolIndex].assets)[0],
+                receivingChain: Object.keys(swapConfig[selectedPoolIndex].assets)[1],
```

Comment:
no more chain name in the test UI ?

Reply:
This was breaking for some reason and I couldn't figure it out. Please help if you can lol.